### PR TITLE
Add jax from scratch notebook

### PR DESCRIPTION
**Pull request by adonath**
This PR adds the notebook that implements a gamma-ray analysis from scratch in jax. Co-authored by @maxnoe and @fabiopintore.

### CODE REVIEW

**Comment by adonath** [comment]
@fabiopintore and @maxnoe I did some cleanup on the notebook and simplify the code a bit. I also removed the scaling factor from loss function and just re-running the notebook with `float64` enabled seemed to work: I got the exact same results for the jax and Minuit minimization.

**Comment by registerrier** [comment]
Thanks @adonath , @maxnoe @fabiopintore !
